pythonforandroid/recipe.py

```diff
@@ -998,7 +1001,8 @@ def build_cython_components(self, arch):
                 info('First build appeared to complete correctly, skipping manual'
                      'cythonising.')
 
-            self.strip_object_files(arch, env)
+            if not self.ctx.build_as_debuggable:
+                self.strip_object_files(arch, env)
```

Comment:
Minor suggestion: just wondering since we're calling this snippet multiple times, maybe we want to make a short helper function/method out of it, e.g.
```python
strip_if_release(self, ctx, env):
    if ctx.build_as_debuggable():
        return
    self.strip_object_files(ctx.archs[0], env, build_dir=ctx.build_dir)
```
Do you think that would make sense?

Reply:
I just looked at this, and since it would need different parameters and essentially just saves one line in two places, it seems to me like this would actually complicate code flow more than worth in saving. But that's just my opinion and as usual I'm happy to add it in if you  really want me to :+1:

Reply:
Yes I'm also fine you keep it this way :+1: